scss/grid/_grid_row.scss

```diff
@@ -19,31 +19,23 @@
   flex-direction: row;
   -ms-flex-wrap: wrap;
   flex-wrap: wrap;
+  width: 100%;
 
   @supports (display: grid){
     display: grid;
     grid-template-columns: repeat(12, 1fr);
     grid-column-gap: 0;
     grid-auto-flow: dense;
-  }
-
-  width: 100%;
 
-  @each $name, $size in $grid-breakpoints {
-    @media (min-width: $size) {
-      @if $name == "md" {
-        @supports (display: grid){
+    @each $name, $size in $grid-breakpoints {
+      @media (min-width: $size) {
+        @if $name == "md" {
           grid-column-gap: map-get($spacers, "sm");
-        }
-      } @else if $name == "lg" {
 
-        @supports (display: grid){
```

Comment:
we're removing nested @supports -- yeah? worth a shot

Reply:
actually no.. the old code was rewriting the `@support` on each media query... I just moved the mediaquery definitions inside the class and not rewrite the `@support` on each one... may be the js sass that fusion use was having some issue with that.